website/static/js/forgotPassword.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * forgot password view model
+ */
+'use srict';
+
+var ko = require('knockout');
+require('knockout-validation');
+
+var $osf = require('osfHelpers');
+
+var ViewModel = function() {
+
+    var self = this;
+
+    self.username = ko.observable('').extend({
+        required: true,
+        email: true
+    });
+
+    self.submit = function() {
```

Comment:
That sounds good to me. Just remind me once #2142 goes in to adjust accordingly.